looper/models.py

```diff
@@ -441,6 +441,8 @@ def __init__(self, config_file, subproject=None,
         self.sheet = None
         self.samples = list()
 
+        self.add_sample_sheet()
```

Comment:
why not add an option in init to shut off the auto-populate functions? then you can isolate things if you need to but the default is to just run everything through on the main init.